Makefile

```diff
@@ -196,3 +196,26 @@ endif
 push: image
 	$(RUNTIME) tag $(IMAGE_PATH) $(IMAGE_PATH):$(TAG)
 	$(RUNTIME) push $(IMAGE_PATH):$(TAG)
+
+codeGenFullPath=$(shell GO111MODULE=on go list -f {{.Dir}} k8s.io/code-generator/cmd/client-gen)
+codegeneratorRoot=$(codeGenFullPath:/cmd/client-gen=)
```

Comment:
huh... what's the = before the closing parenthesis?

Reply:
It's a GNU make substitition, except the substitution is empty, so we cut off the /cmd/client-gen part. See https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_toc.html#TOC61
(btw I took this from the MCO Makefile)

Reply:
btw this dance /might/ be irrelevant if we vendor the codegen